.github/workflows/ci.yml

```diff
@@ -72,12 +72,9 @@ jobs:
             src/rust/target/
           key: ${{ runner.os }}-${{ matrix.PYTHON.VERSION }}-${{ steps.setup-python.outputs.python-version }}-cargo-1-${{ hashFiles('**/Cargo.lock') }}
 
-      - uses: actions/checkout@v3.3.0
-        timeout-minutes: 3
-        with:
-          repository: "google/wycheproof"
-          path: "wycheproof"
-          ref: "master"
+      - name: Clone wycheproof
+        timeout-minutes: 2
```

Comment:
is there a reason not to put the timeout inside?

Reply:
We can’t. I tried 😩